packages/server/src/automations/steps/n8n.ts

```diff
@@ -0,0 +1,143 @@
+import fetch, { HeadersInit } from "node-fetch"
+import { getFetchResponse } from "./utils"
+import {
+  AutomationActionStepId,
+  AutomationStepSchema,
+  AutomationStepInput,
+  AutomationStepType,
+  AutomationIOType,
+  AutomationFeature,
+} from "@budibase/types"
+
+enum Method {
```

Comment:
Could move these into `@budibase/types` and re-use them in the REST integration, just incase these pop up in other places would want to keep from re-writing enums like this all over the place.

Reply:
Added a HttpMethod enum to types, and updated the REST connector.